### ISSUE TITLE
型推論できない時のGenericsの利用はアンチパターンである

### ISSUE THREAD

**Issue by vvakame** [typescript-ninja/typescript-in-definitelyland#12]
https://github.com/borisyankov/DefinitelyTyped/pull/3424#issuecomment-69225572
このへん
考えて盛り込む。

**Comment by johnnyreilly** [comment]
Okay - I cannot read this for the life of me... What is this repo @vvakame?  It's a mystery!

**Comment by vvakame** [comment]
@johnnyreilly do you know comiket?
I was joined it and publish new TypeScript book.
http://techbooster.github.io/c87/#typescript 
It can read here! http://typescript.ninja/typescript-in-definitelyland/

**Comment by johnnyreilly** [comment]
I don't know it.... Lots of Japanese characters that look wonderfully strange to my English eyes!

**Comment by vvakame** [comment]
@johnnyreilly yep. I think english is magical language. english can put all characters on keyboard. It is crazy!
